### ISSUE TITLE
Unable to connect to relative path sessions made w/o zoxide

### ISSUE THREAD

**Issue by GravityShark0** [joshmedeski/sesh#133]
### What happened?

- When I
  `sesh connect` to a session which was made from a relative path that didnt have a zoxide entry before
- I expected
  to connect to the session like normally
- but _ happened
  an error with exit status 1

### Recreation
1. Restart to default state without zoxide
    ```bash
    # Restart to default state
    pkill sesh
    pkill tmux
    # Reset zoxide
    zoxide remove ~/foo
    zoxide remove ~/bar
    # Delete paths
    rmdir ~/foo
    rmdir ~/bar
    # Create paths
    mkdir ~/foo
    mkdir ~/bar
    ```
2. Create session
   ```bash
   cd ~/
   # connects to foo via relative path and detaches creating a new session
   sesh connect --command "tmux detach" foo # or ./foo/
   ```
3. Try to connect to session from a different path
   ```bash
   cd ~/bar
   # (sesh list | head -n 1) should output = foo or _/foo/
   sesh connect $(sesh list | head -n 1) # exit status 1, see relevant log output
   ```

### Notes
- Substituting `foo` in step 2. `sesh connect --command "tmux detach" foo` with its full path `~/foo` or its absolute path `/home/user/foo` makes it works correctly.
- If a zoxide entry already exists for that directory it also works correctly
- I am on Linux and sesh dev was installed via `go install github.com/joshmedeski/sesh@latest` and sesh 1.2.0 from GitHub Releases
- This is my first time making a GitHub issue and I'm also new to go and software development in general
  - I just love what you have made and I thank you for that.

### Version

dev and 1.2.0

### Relevant log output

2024/07/19 10:06:56 unable to connect to "foo": failed to add "/home/gravity/bar/foo" to zoxide: exit status 1

### Reviewed

- [X] Reviewed existing issues and made sure this bug has not already been submitted.
- [ ] I have read the documentation and understand how sesh works before submitting this bug

**Comment by GravityShark0** [comment]
I have made a fork with a hack that sort of works so far in [here](https://github.com/GravityShark0/sesh/tree/relative).
I don't expect it to be merged or anything since it seems v2 is a complete rewrite, but I would like to see it in there.

**Comment by joshmedeski** [comment]
Hey, I do intend on v2 of Sesh being able to handle relative paths, thanks for the flagging this.

**Comment by joshmedeski** [comment]
Can you upgrade and test this and tell me if it's working with Sesh v2?

**Comment by GravityShark0** [comment]
I have upgraded and yes this now works in Sesh v2 from what I've tested.

By the way, I have commented on the previous issue about hiding to already attached sessions session (https://github.com/joshmedeski/sesh/issues/50#issuecomment-2330724158), because `--hide-attached` no longer seems to have any effect in Sesh v2 but still is acceptable as an argument.
I don't know if I should create a new issue about this or reopen the previous one.
But I am closing this one as it seems fine so far.
